bin/plotting/pycbc_plot_vt_ratio

```diff
@@ -1,44 +1,58 @@
 #!/usr/bin/python
 
+"""
+This script compares the sensitivities (VTs) of two searches having a common
+set of injections. It reads two HDF files produced by pycbc_page_sensitivity's
```

Comment:
Common set of injections isn't required, and it's actually assumed that the injections are independent (This makes errors in comparisons look worse)

Reply:
Would "consistent sets" of injections be correct? You still need them to be drawn from the same population, right?

Reply:
that makes sense

Reply:
that makes sense